### DISCUSSION TITLE
Using avocado HTML report for negative filtering

### DISCUSSION THREAD

**Discussion by pevogam** [avocado-framework/avocado#6008] [Q&A] [answered]
He HTML results report plugin of avocado has generally been very useful for us throughout the years for filtering out relevant results by status or test name. Something that I have been wondering more and more with time recently however is if there is any way to perform "negative matching" by typing a word in the search box (possibly a special or a regex syntax?) that could remove certain repeated test fails/errors. Does anyone know if this is easily possible?

**Comment by richtja** [comment]
Hi @pevogam, AFAIK this is not possible right now because avocado uses `datatables 1.10.2` version for HTML report which doesn't have this feature. We could enable the negative search and more by updating the `datatables 2` which has this kind of [features](https://datatables.net/new/2#Search), but right now I can't consider how difficult it would be. If you are interested, you could investigate the possibilities of such update.

**Comment by clebergnu** [comment]
I like the idea, would welcome both the update of datatables and all the features that will come with it.

**Comment by pevogam** [comment]
@lmr How difficult do you think an endeavor like this is going to be? Is it just a matter of simple replacement of a few javascript libraries (plug and play HTML plugin) or perhaps it will have a lot more adapted interfacing involved?

**Comment by pevogam** [comment, marked as answer]
Alright, I went ahead and did this in #6091, I hope you enjoy the advanced search that datatables 2 offers!
